docs/design.md

```diff
@@ -1,96 +1,99 @@
-# Ansible Playbook Bundle (APB) Spec File
+# Design
+
+## Overview
+An [Ansible Playbook Bundle (APB)](https://github.com/fusor/ansible-playbook-bundle)
+borrows several concepts from the [Nulecule](https://github.com/projectatomic/nulecule)
+or [Atomicapp](http://www.projectatomic.io/docs/atomicapp/) project, namely the concept of a short
+lived container with the sole purpose of orchestrating the deployment of the intended application. For the case
+of APB, this short lived container is the APB; a container with an Ansible runtime environment
+plus any files required to assist in orchestration such as playbooks, roles, and extra dependencies.
+Specification of an APB is intended to be lightweight, consisting of several named playbooks and a
+metadata file to capture information such as parameters to pass into the application.
+
+## Workflow
+APB is broken up into the following steps.
+
+  1. [Preparation](#preparation)
+    * [APB init](#apb-initialization)
+    * [Spec File](#spec-file)
+    * [Actions](#actions) (provision, deprovision, bind, unbind)
+  1. [Build](#build)
+  1. [Deploy](#deploy)
```

Comment:
Add another space in front of each of the asterisks (*) otherwise the numbered list doesn't show up right.

What you have is this
```
1. [Preparation](#preparation)                                
  * [APB init](#apb-initialization)                           
  * [Spec File](#spec-file)                                   
  * [Actions](#actions) (provision, deprovision, bind, unbind)
1. [Build](#build)                                            
1. [Deploy](#deploy)                                          
```
Which renders like this:

1. [Preparation](#preparation)                                
  * [APB init](#apb-initialization)                           
  * [Spec File](#spec-file)                                   
  * [Actions](#actions) (provision, deprovision, bind, unbind)
1. [Build](#build)                                            
1. [Deploy](#deploy)                                          

What you need is this:
```
1. [Preparation](#preparation)                                
   * [APB init](#apb-initialization)                           
   * [Spec File](#spec-file)                                   
   * [Actions](#actions) (provision, deprovision, bind, unbind)
1. [Build](#build)                                            
1. [Deploy](#deploy)                   
```
renders like
1. [Preparation](#preparation)                                
   * [APB init](#apb-initialization)                           
   * [Spec File](#spec-file)                                   
   * [Actions](#actions) (provision, deprovision, bind, unbind)
1. [Build](#build)                                            
1. [Deploy](#deploy)